tests/ctst/steps/pra.ts

```diff
@@ -71,6 +71,7 @@ async function installPRA(world: Zenko, sinkS3Endpoint = 'http://s3.zenko.local'
         // prometheusHostname: 'prom.dr.zenko.local', // could be any name, cert will be auto-generated
         prometheusExternalIpsDiscovery: true,
         prometheusDisableTls: true,
+        forceRotateServiceCredentials: true,
```

Comment:
the `force` option should be passed only the `second` install : it shoudl be done autoamtically on the first install (allowing seamless operation), so that is what we want to test in general.

but indeed, we need to pass it on the _second_ install (and only on followup installs)